app/facades/rails_engine_facade.rb

```diff
@@ -0,0 +1,20 @@
+# require './app/poros/merchants.rb'
+
+class RailsEngineFacade
```

Comment:
This is interesting and I like it. We don't necessarily have to have a facade per resource, and given that we aren't calling that many endpoints, it works to have one facade for the `RailsEngineService` versus a separate facade for merchants and items.